news/2022-05-12-osu-talk-event-overcoming-obstacles.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "osu! Talk Event: Overcoming Obstacles"
+date: 2022-05-19 08:00:00 +0000
+---
+
+
+Mark your calendars, set your alarms, and join us for the first ever osu! talk event (OTE), where you can make your voice heard.
+
+
+![](/wiki/shared/news/2022-05-12-osu-talk-event-overcoming-obstacles/ote-newspost-banner.png)
+
+
+Unlike many events that involve guest speakers, OTE is open for anyone to speak -- all you need is a story to share. Each OTE centers around a certain theme. For this OTE, all talks will center around the theme of **overcoming obstacles**.
+
+
+Here's a rundown of how OTE works:
+
+
+- **The event will last 3 days total,** from May 20 - 22, 2022.
+- **The speaker will use a Discord stage channel** in the [official osu!dev Discord server](https://discord.gg/ppy) to give their talk.
+- **Talks will be less than 18 minutes long**. This is the maximum allowed length, not a goal to aim for — even a 3-minute talk can have massive impact.
+- The stories shared do not necessarily have to be about osu!, but because it is an osu!-related event, the speakers are recommended to try connecting their story or the lesson behind their story to osu! in some way.
+
+
+---
+
+
+Below is the talk schedule for the event, including the speaker's username and the name of their talk. All talks will be streamed live on [osu! University's Twitch channel](https://twitch.tv/osuuniversity) and uploaded separately to [osu! University's YouTube channel](https://www.youtube.com/c/osuuniversity) after the event. You can also tune in live through a Discord stage channel in the [official osu!dev Discord server](https://discord.gg/ppy).
+
+__**Friday, May 20, 2022**__
+- 21:30 UTC | [PandaPasBo](https://osu.ppy.sh/users/10262231): "Why expectation ruins your skill in osu"
+- 22:00 UTC | [xootynator](https://osu.ppy.sh/users/3717598): "xooty's left hand training"
+- 22:30 UTC | [Likean00b](https://osu.ppy.sh/users/4860447): "When tournaments become a hindrance"
+- 23:00 UTC | [MakiDonalds](https://osu.ppy.sh/users/11610772): "Break your walls before they break you"
+
+
+__**Saturday, May 21, 2022**__
+- 11:00 UTC | [IryN](https://osu.ppy.sh/users/17909384): "Being hardstuck"
+- 13:00 UTC | [Axiqn](https://osu.ppy.sh/users/21130016): "Focusing on weaknesses"
+- 13:30 UTC | [UserAlsoExists](https://osu.ppy.sh/users/19036931): "The "play less" meta"

```

Comment:
maybe

```suggestion
- 13:30 UTC | [UserAlsoExists](https://osu.ppy.sh/users/19036931): "The 'play less' meta"
```

Reply:
Makes sense as a developer, not sure from an english perspective :D.